zsl/_state.py

```diff
@@ -0,0 +1,14 @@
+_current_app = None
```

Comment:
Multi-application was not the main goal here, only to remove all "app instance" dependent code. This allows further development into a multi application environment. I think the first step should declare _current_app as thread-local variable, but this is for another discussion.